src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/IllegalImportCheck.java

```diff
@@ -77,14 +279,14 @@
     /** The compiled regular expressions for classes. */
     private final List<Pattern> illegalClassesRegexps = new ArrayList<>();
 
-    /** List of illegal packages. */
+    /** Specify packages to reject. See details section below. */
     private String[] illegalPkgs;
 
-    /** List of illegal classes. */
+    /** Specify class names to reject. See details section below. */
```

Comment:
With such reference, javadoc become weird.
I still think that keeping all details in one place, javadoc will looks better.

Reply:
done. reverted to inplace version.